app/src/main/java/com/willowtree/vocable/utils/LocaleUtils.kt

```diff
@@ -0,0 +1,59 @@
+package com.willowtree.vocable.utils
+
+import java.util.*
+
+object LocaleUtils {
```

Comment:
Any static utils classes will need to have tests IMO. Please test it, specially since the app will be pretty reliant on this.